docs/developers/contributing/documentation/index.md

```diff
@@ -88,11 +89,19 @@ cd napari-docs/
 
 Since the API reference documentation (autogenerated from the napari code docstrings) and the example gallery are sourced from the `napari/napari` repository, before you can build the documentation locally, you need:
 
-- a clean `conda` environment;
-- a development installation of napari in that environment. Follow the [contributor guide](dev-installation) for details on how to do this;
-- the napari documentation requirements. Starting at the folder where you cloned the `napari/docs` repository, and after activating your clean `conda` environment, you can install these with `python -m pip install -r requirements.txt`.
+- a clean `conda` environment—remember to activate it!;
+- a from-source, editable installation of napari with the optional `docs` dependencies and a Qt backend. Change directories to where you cloned napari and run, for example:
 
-This setup, with these dependencies, will allow you to preview your document locally as it would appear on `napari.org`. It will also install [Jupyter notebook](https://jupyter.org/) and [jupytext](https://jupytext.readthedocs.io), which you will need to contribute documents containing code or viewer interactions.
+  ```bash
+  python -m pip install -e ".[pyqt, docs]"
+  ```
+  This will use the default Qt backend. For other options, see [the napari installation guide](https://napari.org/stable/tutorials/fundamentals/installation.html#choosing-a-different-qt-backend).
```

Comment:
This should work:

```suggestion
  This will use the default Qt backend. For other options, see [the napari installation guide](#choosing-a-different-qt-backend).
```

Reply:
Actually maybe just 
```suggestion
  This will use the default Qt backend. For other options, see [the napari installation guide](choosing-a-different-qt-backend).
```

with no `#`.